### ISSUE TITLE
Cannot start nginx

### ISSUE THREAD

**Issue by hdavy2002** [KnightSwarm/Minstall#42]
Hi there,

I am using Ubuntu 12.04 on an Ubuntu 12.04 KVM. I am getting this error while starting nginx

Restarting nginx: nginx: [emerg] could not build the types_hash, you should increase either types_hash_max_size: 1024 or types_hash_bucket_size: 32
nginx: configuration file /etc/nginx/nginx.conf test failed

Thanks for your help. I noticed this error on an Openvz container before.

**Comment by maxexcloo** [comment]
I've been having a look and I'm not sure why the issue is occurring.
Have you tried editing the values for types_hash_max_size or types_hash_bucket_size in /etc/nginx/nginx.conf?

**Comment by hdavy2002** [comment]
Yes, I have been editing it from 64 to 512 or 1024, still no effect. I have a new VPS on Ramnode and I am going to try it again.

**Comment by maxexcloo** [comment]
Tell me how it goes, I'm rather confused by this issue...

**Comment by hdavy2002** [comment]
It worked fine on Ramnode. My previous vps was on ChicagoVPS. Recently they had a DOS attack and they are still recovering. I am going to re-install on ChicaoVPS and try once again. Will get back to you after that.

**Comment by hdavy2002** [comment]
I am getting this error after a fresh download of Minstall. This is to with
phpmyadmin script. I only changed the host name thats all. Any ideas?

root@wallbreakers:~/Minstall-2.5.7/extra# ls
benchmark-tools.sh  install-phpmyadmin.sh
root@wallbreakers:~/Minstall-2.5.7/extra# nano install-phpmyadmin.sh
root@wallbreakers:~/Minstall-2.5.7/extra# sh ./install-phpmyadmin.sh
./install-phpmyadmin.sh: 25: ./install-phpmyadmin.sh: Syntax error: "("
unexpect
           ed

##  Help save paper! Do you really need to print this email?

**Comment by maxexcloo** [comment]
Have you tried "bash ./script.sh"?
